scripts/build-plugin.sh

```
@@ -17,9 +17,11 @@ cd ../../..
 
 cp -r frontend/rakuyomi.koplugin/* "$OUT/"
 
-cp backend/target/$TARGET/release/cbz_metadata_reader "$OUT/"
-cp backend/target/$TARGET/release/server "$OUT/"
-cp backend/target/$TARGET/release/uds_http_request "$OUT/"
+if [ "$TARGET" != "none" ]; then
+    cp backend/target/$TARGET/release/cbz_metadata_reader "$OUT/"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)